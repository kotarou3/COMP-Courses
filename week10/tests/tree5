4
20
36
41
48
57
69
75
85
94
