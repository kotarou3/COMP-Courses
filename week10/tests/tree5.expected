Tree:
4
 \
 20
   \
   36
     \
     41
       \
       48
         \
         57
           \
           69
             \
             75
               \
               85
                 \
                 94

#nodes = 10,  #leaves = 1
Infix  : 4 20 36 41 48 57 69 75 85 94 
Prefix : 4 20 36 41 48 57 69 75 85 94 
Postfix: 94 85 75 69 57 48 41 36 20 4 
ByLevel: 4 20 36 41 48 57 69 75 85 94 
